tensorflow/python/ops/standard_ops.py

```diff
@@ -47,6 +47,7 @@
 from tensorflow.python.ops.control_flow_ops import tuple
 from tensorflow.python.ops.control_flow_ops import cond
 from tensorflow.python.ops.control_flow_ops import case
+from tensorflow.python.ops.control_flow_ops import smart_cond
```

Comment:
can you add this to tf.contrib.framework instead of to the core api?

Reply:
Sure, I will import this file in `tf.contrib.framework` and submit a commit later